examples/osm-vector-tiles.js

```diff
@@ -0,0 +1,142 @@
+goog.require('ol.Map');
+goog.require('ol.View');
+goog.require('ol.format.TopoJSON');
+goog.require('ol.layer.VectorTile');
+goog.require('ol.proj');
+goog.require('ol.source.VectorTile');
+goog.require('ol.style.Fill');
+goog.require('ol.style.Stroke');
+goog.require('ol.style.Style');
+
+
+var format = new ol.format.TopoJSON();
+var tileGrid = ol.tilegrid.createXYZ({ maxZoom: 19 });
+var roadStyleCache = {};
+var roadColor = {
+  'major_road': '#776',
+  'minor_road': '#ccb',
+  'highway': '#f39'
+};
+var landuseStyleCache = {};
+var buildingStyle = [
```

Comment:
For future reference, this could also just be a `ol.style.Style` instance instead of an array (and you get a very slight performance gain for never using arrays).